showcase/app/styles/showcase-pages/side-nav.scss

```diff
@@ -8,6 +8,13 @@
 body.components-side-nav {
   $shw-component-side-nav-background: #0c0c0e;
 
+  // Add to examples to constrain width
+  .shw-component-set-max-content-width {
+    width: max-content;
+    height: auto;
+    min-height: 0;
```

Comment:
I added those because I didn't like that the height extended taller than the button which was wrapped even though it wasn't visible. I'll add a comment explaining